docs/data_entry_flow_index.md

```diff
@@ -442,7 +442,6 @@ class TestFlow(config_entries.ConfigFlow, domain=DOMAIN):
         if uncompleted_task:
             return self.async_show_progress(
                 progress_action=progress_action,
-                progress_task=uncompleted_task,
```

Comment:
This was implemented in <https://github.com/home-assistant/core/pull/107668>, and the documentation reflects that.